### PR TITLE
Fix version

### PR DESCRIPTION
**Pull request by Klynger**
#### What problem is this solving?

We had some problem with the version in IO, so this is just for fixing it.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: store/blocks/header/top-menu.json [hunk 1]
```diff
@@ -20,21 +20,6 @@
       }
     }
   },
-  "menu-item#about-us": {
```
Comment (klzns) [inline store/blocks/header/top-menu.json]: Why did you remove this?

Comment (victorhmp) [reply store/blocks/header/top-menu.json]: This block is duplicated

Comment (Klynger) [reply store/blocks/header/top-menu.json]: This is duplicated: https://github.com/vtex-apps/store-theme/blob/master/store/blocks/header/category-menu.json#L155
